index.html

```diff
@@ -345,6 +346,7 @@ <h6 class="h-contact"> contact me</h6>
 
         </form>
       </div>
+    </div>
```

Comment:
- [ ] You did great using the button tag here. :+1: But I think there are still some styling issues you need to update here. 
- [ ] Currently, the position of your button is different from the one in the given design, I'll suggest you update your CSS as replicating the given design is one of the requirements for this milestone. :+1: 

- - YOUR CURRENT DESIGN:
![Screenshot from 2023-06-07 10-24-27](https://github.com/hetrox8/portfolio1/assets/107632780/14f1f7f1-06c9-4916-8271-27086892f314)

- - FIGMA DESIGN:
![Screenshot from 2023-06-07 10-24-12](https://github.com/hetrox8/portfolio1/assets/107632780/97a0707d-abad-4a20-ac4f-c8dfaf0d693a)

- [ ] Also your buttons do not have the interactive properties implemented, You are meant to implement the `hover and pressed` state of your button with the correct background color and text color. I'll suggest you update this too. :+1: :100: 

- - HOVER AND PRESSED STATE FROM DESIGN:
![Screenshot from 2023-06-07 09-16-03](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/107632780/15649450-132c-4ef2-af7e-bba644996e85)